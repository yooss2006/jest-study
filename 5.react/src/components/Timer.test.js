import renderer from "react-test-renderer";
import Timer from "./Timer";

test("초 표시", () => {
  Date.now = jest.fn(() => 123456789);
  const el = renderer.create(<Timer />).toJSON();
  expect(el).toMatchSnapshot();
});
