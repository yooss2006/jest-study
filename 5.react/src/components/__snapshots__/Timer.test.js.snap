// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`초 표시 1`] = `
<div>
  현재 
  36
  초 입니다.
</div>
`;
